doc/contributing/index.rst

```diff
@@ -123,140 +124,19 @@ We provide a pre-commit git hook to prevent accidental commits to the master bra
 Running unit tests
 ------------------
 
-There are two ways to run unit tests for giotto-tda.
+There are two ways to run unit tests for ``giotto-tda``.
 
-1. Using tools and libraries installed directly on your system. The election tool is `pytest`. To install `pytest` just do
+1. Using tools and libraries installed directly on your system. ``giotto-tda`` relies on ``pytest``.
+   To install ``pytest`` just run
 
 .. code-block:: python
 
     pip install pytest
 
-You can use `pytest` on your python code via the following instructions:
+You can use ``pytest`` on your python code via the following instructions:
 
 .. code-block:: python
 
     pytest name_of_your_script.py
 
-2. Using Azure (azure-pipelines.yml) and giotto-tda's CI scripts.
```

Comment:
Do we keep this somewhere (internal knowledge?)

Reply:
Ooops, that's a forgotten link

Reply:
I had misunderstood your question. The link to the yml is still there in the new version.

Reply:
i wasn't clear at all. I meant all of the 'how to generate the docs' part. I

Reply:
Oh! I just removed it as it won't be ready in time. It is in the commit history, but it might be easier for us to make a local copy somewhere.

Reply:
The idea would be to reintroduce this section once automation is solved.